app/uk/gov/hmrc/apisubscriptionfields/model/Responses.scala

```diff
@@ -22,13 +22,14 @@ import play.api.libs.json.{JsObject, Json}
 case class SubscriptionFieldsIdResponse(fieldsId: SubscriptionFieldsId)
 case class SubscriptionFieldsResponse(fieldsId: SubscriptionFieldsId, fields: Fields)
 
+case class FieldsDefinitionResponse(fields: Seq[FieldDefinition])
 
 object ErrorCode extends Enumeration {
   type ErrorCode = Value
 
   val INVALID_REQUEST_PAYLOAD = Value("INVALID_REQUEST_PAYLOAD")
   val UNKNOWN_ERROR = Value("UNKNOWN_ERROR")
-  val SUBSCRIPTION_FIELDS_ID_NOT_FOUND = Value("SUBSCRIPTION_FIELDS_ID_NOT_FOUND")
+  val NOT_FOUND_CODE = Value("NOT_FOUND")
```

Comment:
As discussed before, you need to either use a different `code` or a different `message` for distinguish two cases:
- fields id not found
- id not found